examples/tornjak/README.md

```diff
@@ -42,35 +52,70 @@ You can now access Tornjak at [localhost:3000](http://localhost:3000).
 
 See [values.yaml](./values.yaml) for more details on the chart configurations to achieve this setup.
 
-## Tornjak and Ingress with ingress-nginx
+## Deploy Tornjak with ingress-nginx
 
-Update examples/production/example-your-values.yaml with your information, most importantly, trustDomain.
+Update your-values.yaml with your information, most importantly, trustDomain, and redeploy.
 
 ```shell
-helm upgrade --install --namespace spire-server spire charts/spire \
---values examples/production/values.yaml \
+helm upgrade --install -n spire-mgmt spire spire \
+--repo https://spiffe.github.io/helm-charts-hardened/ \
 --values examples/tornjak/values.yaml \
 --values examples/tornjak/values-ingress.yaml \
 --set global.spire.ingressControllerType=ingress-nginx \
---values examples/production/example-your-values.yaml \
---render-subchart-notes --debug
+--values your-values.yaml \
+--render-subchart-notes
 ```
 
 ## Tornjak and Ingress on Openshift
 
-When deploying on Openshift, follow the deployment setup as described in
-[Openshift README](../openshift/README.md)
+Obtain the OpenShift Apps Subdomain for Ingress
+We will be using the OpenShift application subdomain during our deployment, so let's capture it now and create environment variable by executing the following command:
 
-Then just add Openshift specific configuration to the above command:
+```shell
+export appdomain=$(oc get cm -n openshift-config-managed  console-public -o go-template="{{ .data.consoleURL }}" | sed 's@https://@@; s/^[^.]*\.//')
+echo $appdomain
+```
+
+We can now use this variable during the installation of the SPIRE Helm charts.
 
 ```shell
---values examples/openshift/openshift-values.yaml
+helm upgrade --install -n spire-mgmt spire spire \
+--repo https://spiffe.github.io/helm-charts-hardened/ \
+--set global.openshift=true \
+--set global.spire.trustDomain=$appdomain \
+--set spire-server.ca_subject.common_name=$appdomain \
+--set spire-server.ingress.host=spire-server.$appdomain \
```

Comment:
Everything is looking pretty good now.

Do we still need this bit though? Setting these values is already done, I think, in the base install instructions. Without these, its basically the same instructions for "Tornjak and Ingress" on Vanilla? Think they could be combined into one?

Reply:
I modified the text a little and since the main doc (spiffe.io) only briefly mentions the Openshift, I provided a useful example for the deployment